lib/assertions.js

```diff
@@ -55,33 +55,35 @@ const identities = {
   ],
 }
 
-const makeCorpPass = ({ NRIC, UEN }) => render(
+const corpPassNric = process.env.MOCKPASS_NRIC || identities.corpPass[0].NRIC
+const uen = process.env.MOCKPASS_UEN || identities.corpPass[0].UEN
+
+const makeCorpPass = (source = { NRIC: corpPassNric, UEN: uen }) => render(
   TEMPLATE,
   {
     issueInstant: moment.utc().format(),
-    name: UEN,
-    value: base64.encode(render(corpPassTemplate, { NRIC, UEN })),
+    name: source.UEN,
+    value: base64.encode(render(corpPassTemplate, source)),
   }
 )
 
-const makeSingPass = (NRIC) => render(TEMPLATE,
+const nric = process.env.MOCKPASS_NRIC || identities.singPass[0]
+const recipient = process.env.SERVICE_PROVIDER_RECIPIENT || "http://localhost:8080/saml/SSO"
+
+const makeSingPass = (NRIC = nric, RECIPIENT = recipient) => render(
+  TEMPLATE,
   {
     name: 'UserName',
     value: NRIC,
     issueInstant: moment.utc().format(),
+    recipient: RECIPIENT,
```

Comment:
You must have made changes to the assertion template, yet it's not in the PR. 
* Would you be able to include it please? 
* Wouldn't the presence of `recipient` also need to be reflected in `makeCorpPass()` too?

Reply:
I've included the xml files in the PR and added recipient to makeCorpPass()